### PR TITLE
Add color feedback to transaction text input

### PR DESCRIPTION
**Pull request by RobPando**

### CODE REVIEW

File: app/assets/javascripts/components/input_field.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+$(document).ready(function() {
```
Comment (pablocrivella) [inline app/assets/javascripts/components/input_field.js]: Use coffeescript or ES6

**Comment by RobPando** [comment]
@pablocrivella Updated!

File: app/assets/javascripts/components/input_field.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+document.addEventListener('DOMContentLoaded', function() {
+  const transactionFields = Array.from(document.querySelectorAll('.number-field'))
```
Comment (pablocrivella) [inline app/assets/javascripts/components/input_field.js]: Maybe this should a more specific selector or a data attribute. What do you think?

File: app/assets/javascripts/components/input_field.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+  transactionFields.forEach((transactionField) => {
+    transactionField.addEventListener('keyup', function() {
+      const value = parseInt(this.value)
+
+      if (value > 0) {
```
Comment (pablocrivella) [inline app/assets/javascripts/components/input_field.js]: Maybe you could clean this up using, toggle or replace:

https://developer.mozilla.org/en-US/docs/Web/API/Element/classList

Comment (RobPando) [reply app/assets/javascripts/components/input_field.js]: replace 👍

Comment (RobPando) [reply app/assets/javascripts/components/input_field.js]: NVM, it does not work if the old class is not present apparently, and using toggle would result in the same amount of lines. I will keep as is.

**Comment by RobPando** [comment]
@pablocrivella Updated!

**Pull request merged**